src/main/native-package/m4/tcnative.m4

```diff
@@ -281,7 +281,7 @@ dnl
 dnl Configure for the detected openssl toolkit installation, giving
 dnl preference to "--with-ssl=<path>" if it was specified.
 dnl
-AC_DEFUN(TCN_CHECK_SSL_TOOLKIT,[
+AC_DEFUN([TCN_CHECK_SSL_TOOLKIT],[
 OPENSSL_WARNING=
 AC_MSG_CHECKING(for OpenSSL library)
 AC_ARG_WITH(ssl,
```

Comment:
@nmittler let us fix this in another PR as you will also need to fix the 'bootstrap' branch